src/containers/CredentialShareModal.js

```diff
@@ -0,0 +1,174 @@
+import React, {useEffect} from "react";
+import {Alert, Table, Button, ControlLabel, FormControl, FormGroup, Modal} from "react-bootstrap";
+import {useAsync, useAsyncFn} from "react-use";
+
+function parseInfoFromToken(token) {
+    try {
+        const payload = JSON.parse(atob(token.split('.')[1]))
```

Comment:
@Yengas please use this static method/helper: https://github.com/affinityproject/affinidi-core-sdk/blob/master/sdk/core/src/CommonNetworkMember.ts#L1516 to parse JWT